pkg/csi_driver/controller.go

```diff
@@ -107,6 +120,7 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 			return nil, status.Error(codes.Internal, err.Error())
 		}
 	}
+	s.config.mutex.Unlock()
```

Comment:
It's better to use `defer s.config.mutex.Unlock()` after `Lock()` so you don't miss unlocking when you return early from an error